website/src/CONTRIBUTING.md

```diff
@@ -0,0 +1,309 @@
+# Contributing to OpenEBS Website
+
+## Add a new blog post
+
+* Create a markdown file and add your blog content along with the metadata in    the following format
+```
+---
+title: Title of the Blog
+author: Author name
+author_info: Author information
+tags: All the tags with space in between (Example: openebs localpv)
+date: Date of publication in dd-mm-yyyy format
+---
+ 
+Blog content in the markdown format goes here…
+
+```
+Reference for text formatting in markdown https://guides.github.com/features/mastering-markdown/
+* Name of the markdown file should be the title of the blog converted to all small letters with hyphen(-) separation between the words.
+For example, If the title of the blog is: Title of the Blog
+Then, markdown file name will be title-of-the-blog.md 
+* Add this markdown file to website/src/blogs
+* The main blog image should be named as title-of-the-blog.png and add this file to website/public/Images/blog
+* If you are a new author, add your photograph with file name as author-name.png to website/public/Images/blog/authors. Make sure the name provided in the author section of the metadata of the blog matches with this filename
+
+
+## Add a new  event
+
+
+* Go to website/src/resources/events.json and make the new entry in the events.json file by adding another json object literal after the last event.
+* Format for the event is as follows
+
+```
+{
+       "id": Unique key identifying the event (+ve number),
+       "date": "Date of commencement of the event in yyyy-mm-dd format",
+       "title": "Event name",
+       "description": "Description of the event",      
+       "buttonText": "Register",
+       "buttonLink": "https://openebs.io/"
+}
+```
+
+
+## Add a new testimonial/adopter
+
+* Go to https://github.com/openebs/openebs/blob/master/ADOPTERS.md and grab some of the testimonials from the list and add them to the websites as shown below.
+
+* For an organization testimonial make a new entry in the website/src/adopters.md. file.
+https://github.com/openebs/website/blob/revamp-3.0/website/src/adopters.md
+
+![testimonial table](/public/Images/guidelines/image9.png)
```

Comment:
I believe that `Images` is changed to `images` in this PR (https://github.com/openebs/website/pull/121).